### PR TITLE
fix typos in ZTP docs

### PR DESCRIPTION
**Pull request by aireilly**
Merge to main, CP to 4.10+

### CODE REVIEW

File: modules/ztp-acm-adding-images-to-mirror-registry.adoc [hunk 1]
```diff
@@ -36,7 +36,7 @@ $ export ROOTFS_IMAGE_NAME=<rootfs_image_name> <2>
 +
 [source,terminal]
 ----
-$ export OCP_VERSION=<ocp_version> <2>
+$ export OCP_VERSION=<ocp_version> <3>
```
Comment (bergerhoffer) [inline modules/ztp-acm-adding-images-to-mirror-registry.adoc:39]: I didn't even know that you could do callouts like this, across multiple code blocks and have them all together at the end :thinking:

**Review by bergerhoffer — COMMENTED** [review]
LGTM! Only comment is just stating something that I learned today :slightly_smiling_face: 

Also, technically I think even these small ones should be going to peer review first, based on the defined process. But we could always chat with Kathryn to see if they think we could skip the peer review step for small typos or not.

**Comment by bergerhoffer** [comment]
@aireilly The CP to 4.10 failed, can you please do a manual CP?

**Comment by aireilly** [comment]
> @aireilly The CP to 4.10 failed, can you please do a manual CP?

Oops! Got ahead of myself. I need to merge the 4.10 updates first here: https://github.com/openshift/openshift-docs/pull/53026 I think I will just CP this change into that PR for simplicity's sake.

**Comment by bergerhoffer** [comment]
Okay, I'll mark this PR as just 4.11+ then. Thanks!

**Pull request merged**